workflow-examples/src/main/java/com/redhat/parodos/examples/simple/task/RestAPIWorkFlowTask.java

```diff
@@ -73,7 +73,7 @@ public List<WorkFlowTaskParameter> getWorkFlowTaskParameters() {
 						.type(WorkFlowTaskParameterType.URL).build(),
 				WorkFlowTaskParameter.builder().key(PAYLOAD_KEY)
 						.description("Json of what to provide for data. (ie: 'Hello!')").optional(false)
-						.type(WorkFlowTaskParameterType.PASSWORD).build(),
+						.type(WorkFlowTaskParameterType.TEXT).build(),
```

Comment:
What is the reason to change type here?

Reply:
json payload is not a password type at all.